storage/migrations/01_oasis_3_consensus.up.sql

```diff
@@ -101,7 +101,7 @@ CREATE TABLE oasis_3.nodes
   -- For those cases, (id, entity_id) is not a foreign key into oasis_3.claimed_nodes.
   -- Similarly, an entity can un-claim a node after the node registered, but the node can remain be registered for a while.
   entity_id  base64_ed25519_pubkey NOT NULL REFERENCES oasis_3.entities(id),
-  expiration BIGINT NOT NULL,
+  expiration UINT63 NOT NULL,
```

Comment:
Optional:
```suggestion
  expiration UINT63 NOT NULL,  -- The epoch in which this node expires.
```
(I was wondering if it was a timestamp, or a round, or something else)

Reply:
Added!